src/components/playback-bar/index.js

```diff
@@ -129,8 +130,40 @@ const PlaybackBar = ({ isLoading, onPreload, onSeek, onChange }) => {
     setSelectedIncrement(
       playbackIncrementForSelectedDuration(selectedTimeRange)
     );
+    setNerdletState((prevState) => ({
+      flow: {
+        ...prevState.flow,
+        pb_selectedTimeRange: selectedTimeRange,
+      },
+    }));
   }, []);
 
+  const playheadIncrementChangeHandler = useCallback((item) => {
+    setSelectedIncrement(item);
+    setNerdletState((prevState) => ({
+      flow: {
+        ...prevState.flow,
+        pb_selectedIncrement: item,
+      },
+    }));
```

Comment:
Preferably set the playback selected increment in `nerdletState` here, and have a `useEffect` to handle the state change.